### ISSUE TITLE
[ Bug report ] Some comments are stripped during TS -> JS conversion

### ISSUE THREAD

**Issue by lorensr** [temporalio/documentation#1347]
## Required

**What is the issue?**

Some comments are stripped during TS -> JS conversion

![image](https://user-images.githubusercontent.com/251288/181370990-8d0805c2-950d-4a85-ac17-12a1857aa039.png)

![image](https://user-images.githubusercontent.com/251288/181370980-41b97d55-a2b2-4ef3-851c-f9f6ce45a7e6.png)

**How can we reproduce the issue?**

https://docs.temporal.io/application-development/foundations?lang=js#set-task-queue

**What is the expected behavior?**

Not stripped

**Comment by vkarpov15** [comment]
@lorensr I took a look and @napcs is right: this is something that the TypeScript compiler does, and not something we can easily configure away.

I found a couple of alternative workarounds:

1) Trailing comments on object properties get stripped out, but leading properties don't. For example, take a look at this [TypeScript playground](https://www.typescriptlang.org/play?ssl=13&ssc=1&pln=14&pc=1&downlevelIteration=false&removeComments=false#code/PTAEwFwUwWwBwPYCcCGAbYBnJBjYB3ZAayiQDoJMAoAS3mQlAG9QB1Y00AX1ADMkEMUAHIAAtHqo0NBAQ5JhAblqTGAKlApMmnBBoA3Gnqjb+gkWWApdBozRNKqVJAFcAdgAoAlGRwoIOAAWHh6kSF6gALwAfKA4CG6YCGhQZGgIAOahSOFeylRaAJ5uOHzuNgmgrp4RTFSgcQmYjIRIJEhRmvgoRmzyvkhQ-lAedQ0NIHwICABGKEj1461EvOn4mAAK-oEAXFVQAI4uNINkg0lo+iPClsurCOvCXgA0ixNgACYnULpohaB0RBIaAfAFuUAAOQQH1SACtqOMdHpDMZMK9EZN4jAYFA3BA3qAIFoiABFFxQcl7YQQFwQZA0dDCdHcPKLFDdXrLUhndzeRTgMBYnF4qhcKhAA). The following code retains comments after TypeScript compilation. We would just have to change the format of our comments. This would be the easiest solution; no extra code required, just write comments slightly differently.

```ts
  const worker = await Worker.create({
    // foobar
    workflowsPath: require.resolve('./workflows'),
    // directly imported in Node.js
    activities,
    // comment
    taskQueue: 'tutorial',
  });
```

2) We can write a TypeScript transformer like in [this SO](https://stackoverflow.com/questions/50153634/typescript-compiler-api-emit-comment-before-typescript-constructor-and-properti). In theory we should be able to fix this with a transformer, but I'm not entirely certain. This approach would also be more involved from a code perspective. But we wouldn't have to change our comment syntax.
